index.html

```diff
@@ -21,6 +21,7 @@
               d="M17 15.586l-3.542-3.542A7.465 7.465 0 0 0 15 7.5 7.5 7.5 0 1 0 7.5 15c1.71 0 3.282-.579 4.544-1.542L15.586 17 17 15.586zM2 7.5C2 4.467 4.467 2 7.5 2 10.532 2 13 4.467 13 7.5c0 3.032-2.468 5.5-5.5 5.5A5.506 5.506 0 0 1 2 7.5z" />
           </svg>
         </button>
+        <label for="search_field" class="visually-hidden">Поиск</label>
```

Comment:
```suggestion
        <label for="search_field" class="visually-hidden">Поиск по сайту</label>
```
согласись, так удобнее звучит для пользователя то, что этот поиск делает?

Reply:
соглашусь